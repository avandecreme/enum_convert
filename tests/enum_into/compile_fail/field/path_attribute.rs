use enum_convert::EnumInto;

#[derive(EnumInto)]
#[enum_into(Target)]
enum Source {
    Data {
        #[enum_into] // Should be #[enum_into(Target::a)]
        x: i32,
    },
}

enum Target {
    Data { a: i64 },
}

fn main() {}
