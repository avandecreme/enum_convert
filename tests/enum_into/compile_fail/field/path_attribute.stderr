error: expected a list of field names, for example #[enum_into(Target1::field_a, Target2::field_b)]
 --> tests/enum_into/compile_fail/field/path_attribute.rs:7:9
  |
7 |         #[enum_into] // Should be #[enum_into(Target::a)]
  |         ^
